environment.yml

```diff
@@ -3,177 +3,17 @@ channels:
   - conda-forge
   - bioconda
   - defaults
+  - pkgs/main
```

Comment:
Looks much better :+1: 

What does this line do? Never seen it before in an `environment.yml`.

Reply:
Also, did you try and build this environment locally and test that it works with the pipeline using `-profile conda`?

Reply:
Removed the line, this was a leftover from trying to fix the linting issues where exoisting conda versions are not correctly identified (https://travis-ci.com/nf-core/denovohybrid/jobs/226690048#L1023).

I also added back a few tools that i removed but were needed